src/fields/AbstractField.js

```diff
@@ -172,6 +202,8 @@ class AbstractField extends React.Component {
   }
 
   onFocus = () => {
+    const {onFocus} = this.props;
+    onFocus && onFocus();
```

Comment:
use if statement here, ` onFocus && onFocus();` `&&`can cause unpredictable behaviour in some cases